packages/account-v2/src/constants/paymentMethodsConfig.tsx

```diff
@@ -0,0 +1,291 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+import React from 'react';
+import {
+    DerivLightWalletIcon,
+    PaymentMethodAdvcashBrandDarkIcon,
+    PaymentMethodAdvcashBrandIcon,
+    PaymentMethodAstropayBrandDarkIcon,
+    PaymentMethodAstropayBrandIcon,
+    PaymentMethodBeyonicBrandIcon,
+    PaymentMethodBoletoBrandIcon,
+    PaymentMethodBoletoWhiteIcon,
+    PaymentMethodJetonBrandIcon,
+    PaymentMethodMastercardBrandDarkIcon,
+    PaymentMethodMastercardBrandIcon,
+    PaymentMethodNetellerBrandIcon,
+    PaymentMethodNetellerWhiteIcon,
+    PaymentMethodOnlinenairaBrandIcon,
+    PaymentMethodPixBrandIcon,
+    PaymentMethodPixWhiteIcon,
+    PaymentMethodSkrillBrandIcon,
+    PaymentMethodSkrillWhiteIcon,
+    PaymentMethodSticpayBrandIcon,
+    PaymentMethodVisaBrandIcon,
+    PaymentMethodVisaWhiteIcon,
+    PaymentMethodWebmoneyBrandIcon,
+    PaymentMethodZingpayBrandDarkIcon,
+    PaymentMethodZingpayBrandIcon,
+} from '@deriv/quill-icons';
+import { Text } from '@deriv-com/ui';
+
+const iconSize = { height: '40px', width: '64px' };
+
+export const PAYMENT_METHOD_IDENTIFIER = Object.freeze({
+    accountID: 'account_id',
+    accountNumber: 'account_number',
+    bankAccountNumber: 'bank_account_number',
+    cardNumber: 'card_number',
+    email: 'email_address',
+    mobileNumber: 'mobile_number',
+    userID: 'user_id',
+});
+
+export const getPaymentMethodsConfig = () => ({
+    advCash: {
+        icon: {
+            dark: <PaymentMethodAdvcashBrandDarkIcon {...iconSize} />,
+            light: <PaymentMethodAdvcashBrandIcon {...iconSize} />,
+        },
+        identifier: 'email_address',
```

Comment:
cant we use PAYMENT_METHOD_IDENTIFIER.email

Reply:
Thanks. I missed it